### DISCUSSION TITLE
Is statedump still available in community edition?

### DISCUSSION THREAD

**Discussion by zdam** [Keyfactor/ejbca-ce#693] [General]
I'm having trouble finding documentation for statedump. 
Does it still exist as a tool in the community edition?

**Comment by primetomas** [comment]
Statedump was never available in Community Edition it was/is a separate add on tool.

**Comment by zdam** [comment]
Could you point to where it is please?

**Reply by primetomas** [reply]
If you have a support contract with Keyfactor, create a support ticket to get access. If not you can get in contact for EJBCA Enterprise [here](https://www.keyfactor.com/contact-us/), or contact me and I can point you in the right direction.
